client/mapping-sync/src/sql/mod.rs

```diff
@@ -39,7 +47,12 @@ pub enum WorkerCommand {
 	/// Index leaves.
 	IndexLeaves(Vec<H256>),
 	/// Index the best block known so far via import notifications.
-	IndexBestBlock(H256),
+	/// When `reorg_data` is Some, emits a pubsub notification with reorg info after indexing.
+	/// When `reorg_data` is None, emits a simple new block notification.
+	IndexBestBlock {
+		block_hash: H256,
+		reorg_data: Option<ReorgData>,
+	},
```

Comment:
We should not need reorg_data in `IndexBestBlock`. At this point we cannot yet emit re-org information. It should be done in `Canonicalize`